### ISSUE TITLE
Optimize numpy calls

### ISSUE THREAD

**Issue by arthurpaulino** [trajminer/trajminer#58]
Calling `numpy.function(obj)` is usually slower than calling `obj.function()`, if possible. For instance:

```
$ python -m timeit -s "import numpy as np; m=np.ones((1000, 1000))" "np.transpose(m)"
500000 loops, best of 5: 419 nsec per loop
$ python -m timeit -s "import numpy as np; m=np.ones((1000, 1000))" "m.transpose()"
2000000 loops, best of 5: 128 nsec per loop
```

I was checking out your repo and I saw that you used the slower call a few times (e.g.: [np.transpose](https://github.com/trajminer/trajminer/search?q=np.transpose&unscoped_q=np.transpose)). So I think you'd like to change those.

**Comment by lucaspetry** [comment]
Yep, thank you for pointing that out! Also, feel free to submit a PR!

**Comment by arthurpaulino** [comment]
> Also, feel free to submit a PR!

Thanks! I just need to get a few things done before I can help with code.

**Comment by lucaspetry** [comment]
Fixed in #61
